runtime/Cargo.toml

```diff
@@ -12,8 +12,17 @@ description = "Provides the deno runtime library"
 [features]
 # "fake" feature that allows to generate docs on docs.rs
 docsrs = []
-# feature that modifies the snapshot to allow extending it
+# A feature that disables creation of startup snapshot during in the build script.
+dont_create_runtime_snapshot = []
+# A feature that changes how startup snapshot is generated, that allows
+# extending it in embedder crates.
 snapshot_from_snapshot = []
+# A feature that disables embedding of the JavaScript source files in the binary.
+# With this feature enabled, the sources must be consumed during build time,
+# by creating a startup snapshot.
+include_js_files_for_snapshotting = [
+  "deno_core/include_js_files_for_snapshotting",
+]
```

Comment:
I think this can be simplified:

```toml
default = ["snapshot"]

# Snapshot the internal runtime code into a V8 snapshot, instead of
# bootstrapping isolates from scratch at runtime.
snapshot = ["deno_core/include_js_files_for_snapshotting"]
# A feature that changes how startup snapshot is generated, that allows
# extending it in embedder crates.
snapshot_from_snapshot = []

# "fake" feature that allows to generate docs on docs.rs
docsrs = []
```

Reply:
Discussed offline, we'll tackle this after 1.31, as we can't get the size to 82Mb (but 84Mb) with this change.